data/strings/types_strings.txt

```diff
@@ -18193,17 +18193,98 @@
     pl = Odkryte skały
     pt = Zona rochosa
     ro = Piatră goală
-    ru = Скалы
+    ru = Каменная порода
     sk = Holá skala
     sv = Kala stenar
     sw = Mwamba tupu
     th = เปลือยร็อค
     tr = Çıplak Kaya
-    uk = Скелi
+    uk = Кам'яна порода
     vi = Đá trần
     zh-Hans = 岩石
     zh-Hant = 岩石
 
+  [type.natural.shingle]
+    en = Shingle
+    af = Gordelroos
+    ar = تقرير
+    az = Şingle
+    be = Галька
+    bg = Камъчета
+    ca = Teula
+    cs = Oblázky
+    da = Småsten
+    de = Kieselsteine
+    el = Βότσαλα
+    es = Guijarros
+    es-MX = Guijarros
+    et = Pebbles
+    fa = گزارش
+    fi = Pebbles
+    fr = Galets
+    he = להגיש תלונה
```

Comment:
Two options:

literally "stones":
```suggestion
    he = אבנים
```
or

literally "gravel":
```suggestion
    he = חצץ
```


Maybe there's a more accurate technical term that I'm not familiar with. Looking this up is really hard because the English word "shingle" has too many meanings. The few answers I did find just say to use the word for gravel.

Reply:
Gravel could be sharp, shingle/pebbles is supposed to be rounded.

Reply:
AFAIK there is no word for that in Hebrew. My intuition is to call it אבנים (stones), but I don't know if that's just me or something most people would agree with.

Reply:
perhaps אבנים חלקות (smooth stones), but that sort of implies they're slippery which is also not the case...